server/data/activitiesApiClient.ts

```diff
@@ -240,13 +244,12 @@ export default class ActivitiesApiClient extends AbstractHmppsRestClient {
   }
 
   async updateAttendances(attendanceUpdates: AttendanceUpdateRequest[], user: ServiceUser): Promise<void> {
-    return this.put(
-      {
-        path: '/attendances',
-        data: attendanceUpdates,
-      },
-      user,
-    )
+    return this.put({
+      path: '/attendances',
+      data: attendanceUpdates,
+      authToken: user.token,
```

Comment:
I'm not sure the user token can be used for attendance. It's because it can result in case notes being added, and this requires a role that is not usually given to users - only to our client.  Try with an attendance that needs a case note to confirm this though?